example/spec/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -173,6 +173,23 @@ describe('The S3 Ingest Granules workflow', () => {
     expect(workflowExecution.status).toEqual('SUCCEEDED');
   });
 
+  it('pulls a provider from the list of providers', async () => {
```

Comment:
I could be wrong, but it seems like these tests belong more in `APISuccessSpec`, since they're not really specific to the `IngestGranule` workflow. We could add basic setup to `APISuccessSpec` to create provider/collection via the API and move these specs there

Reply:
Yeah I do have them in APISuccessSpec. This was just kind of an experiment, I wanted to make sure I could find the exact collection and provider that you created in your PR. But I could change this. I can try calling Get for the specific provider and collection that was created.

Reply:
Yeah, I was just thinking that the test for getting a specific provider/collection that was created isn't really specific to `IngestGranule` and could be done in `APISuccessSpec` if we replace `addCollection` with `apiTestUtils.addCollectionApi`, etc

Reply:
Nevermind. I see that [@laurenfrederick's PR is going to remove `APISuccessSpec`](https://github.com/nasa/cumulus/pull/611/files#diff-3b2cbf1fbd146e3574423205d0ddc063), so no point in moving anything there